### PR TITLE
move minimize simulator button under simulator

### PR DESCRIPTION
**Pull request by pelikhan**
This is a preparation of the pairing button. It is a UI change that should be shipped with micro:bit - pairing button will come later.
* only impacts "computer" size
collapsebutton

### CODE REVIEW

**Comment by samelhusseini** [comment]
The plan here was to bump the entire editor to the right and show a big unminimize button to the left. ie: move it away from the download button entirely

**Comment by pelikhan** [comment]
Correct but that requires more work. For now, I would like to get the change of moving the minize button in the simulator because it will impact most screenshots of the bbc microbit editor. The collapsed state polish can be worked on later.

File: webapp/src/simtoolbar.tsx [hunk 1]
```diff
@@ -99,8 +106,11 @@ export class SimulatorToolbar extends data.Component<SimulatorProps, {}> {
             </div>
             <div className={`ui icon tiny buttons ${isFullscreen ? 'massive' : ''}`} style={{ padding: "0" }}>
                 {audio ? <sui.Button key='mutebtn' className={`mute-button ${isMuted ? 'red' : ''}`} icon={`${isMuted ? 'volume off' : 'volume up'}`} title={muteTooltip} onClick={this.toggleMute} /> : undefined}
+            </div>
+            <div className={`ui icon tiny buttons ${isFullscreen ? 'massive' : ''}`} style={{ padding: "0" }}>
+                {!isFullscreen ? <sui.Button key='collapsebtn' className={`collapse-button`} icon={`icon toggle left`} title={collapseTooltip} onClick={this.toggleSimulatorCollapse} /> : undefined}
                 {fullscreen ? <sui.Button key='fullscreenbtn' className={`fullscreen-button`} icon={`xicon ${isFullscreen ? 'fullscreencollapse' : 'fullscreen'}`} title={fullscreenTooltip} onClick={this.toggleSimulatorFullscreen} /> : undefined}
             </div>
-        </aside>;
+        </aside >;
```
Comment (samelhusseini) [inline webapp/src/simtoolbar.tsx]: nit: is this whitespace expected?

Comment (pelikhan) [reply webapp/src/simtoolbar.tsx]: Added by tslint

**Pull request merged**